src/rules/deprecation.ts

```diff
@@ -132,7 +131,7 @@ function isDeclaration(
       // Prevent considering initializer, extends, or implements to be declaration
       return parent.id === id;
 
-    case 'ClassProperty':
+    case 'PropertyDefinition':
```

Comment:
this change is backwards compatible with eslint 7?

Reply:
Hmm, good question.
Now that I look at it I believe it's not and will only work with eslint 8+...

Reply:
Should we just keep both cases instead of replacing the old one?

Reply:
This was fixed in #41 and released in v1.3.2!